Source/Dronecode-SDK-Swift/Dronecode_SDK_SwiftIntegrationTests/TelemetryTest.swift

```diff
@@ -112,4 +112,17 @@ class TelemetryTest: XCTestCase {
             XCTFail("GroundSpeedObservable is expected to receive at least one event, but it did not!")
         }
     }
+
+    /* TEST FAILS
```

Comment:
No need to comment out the integration tests, anyway we don't run them in a CI right now.